README.md

```diff
@@ -18,29 +18,55 @@ This will also install some Git pre-commit hooks that check any modified files a
 
 ### Regular development
 
-To run all tests and linters:
+TL;DR: All checks are run with:
 ```sh
 npm t
 ```
+#### Dev Entry point
 
-Several npm scripts are available to only run a subset of these, such as:
+To run the development entry point:
```

Comment:
Would something like “(aka the dev server)” make sense here? (That’s the term we use in the cypress section, and I think it might help to mention it here too.)

Reply:
makes sense :+1: